cmd/gendeepcopy/deep_copy.go

```diff
@@ -63,7 +63,7 @@ func main() {
 		vals := strings.Split(overwrite, "=")
 		generator.OverwritePackage(vals[0], vals[1])
 	}
-	for _, knownType := range api.Scheme.KnownTypes(knownVersion) {
+	for _, knownType := range api.Scheme.KnownTypes(api.Group, knownVersion) {
```

Comment:
same update the description of version to say it includes group as well.
We can call the variable groupVersion